ansible/idr-microservices.yml

```diff
@@ -0,0 +1,131 @@
+# IDR initial test of microservices
+
+
+- hosts: "{{ idr_environment | default('idr') }}-database-hosts"
+# Load hostvars for subsequent playbooks
+
+
+- hosts: "{{ idr_environment | default('idr') }}-omero-hosts"
+
+  # pre_tasks:
+  #   - debug: var=inventory_hostname
+  #   - debug: msg="{{ hostvars[inventory_hostname] | to_yaml }}"
+  #   - fail: msg=stopping
+
+  roles:
+
+    - role: ome.docker
+      docker_use_ipv4_nic_mtu: True
+
+  tasks:
+
+    - name: Install docker-python
+      become: true
+      yum:
+        name: docker-python
+        state: present
+
+    - name: Get IPs of hosts
+      set_fact:
+        omero_host_ip: >-
+          {{ hostvars[inventory_hostname]['ansible_' + (idr_net_iface |
+             default('eth0'))]['ipv4']['address']
+          }}
+        omero_db_host_ansible: >-
+          {{ hostvars[groups[idr_environment |
+             default('idr') + '-database-hosts'][0]][
+             'ansible_' + (idr_net_iface | default('eth0'))]['ipv4']['address']
+          }}
+
+    - name: omero-ms config directories
+      become: true
+      file:
+        path: /etc/omero-ms-{{ item }}
+        state: directory
+      with_items:
+        - thumbnail
+        - imageregion
+        - zarr
+
+    - name: omero-ms config files
+      become: true
+      template:
+        src: templates/omero-ms-{{ item }}-config.yml.j2
+        dest: /etc/omero-ms-{{ item }}/config.yml
+      with_items:
+        - thumbnail
+        - imageregion
+
+    - name: omero-ms custom entrypoints
+      become: true
+      template:
+        src: templates/omero-ms-{{ item }}-entrypoint.sh.j2
+        dest: /etc/omero-ms-{{ item }}/omero-ms-{{ item }}-entrypoint.sh
+        mode: u=rwx,g=rx,o=rx
+      with_items:
+        - imageregion
+        - zarr
+
+    - name: Run docker omero-ms-thumbnail
+      become: true
+      docker_container:
+        image: openmicroscopy/omero-ms-thumbnail:0.5.3
```

Comment:
Latest release of `omero-ms-thumbnail` is 0.5.7 https://hub.docker.com/r/openmicroscopy/omero-ms-thumbnail/tags